### ISSUE TITLE
edit name

### ISSUE THREAD

**Issue by ubiteczla** [traccar/traccar-web#1095]
**Is your feature request related to a problem? Please describe.**
the ability for users to rename the device to their liking

**Describe the solution you'd like**
a checkbox, possibly in the device connections tab, which shows the current name and you can rewrite the name as desired

**Describe alternatives you've considered**
a button in statuscard that leads to a lighter version of devicespage where only the name, photo and category are displayed, which are values ​​that the end user always requests to change

**Comment by tananaev** [comment]
Users can already edit the device name.

**Comment by ubiteczla** [comment]
Yes, you are right, I forgot to explain a bit, in common use the vast majority of users are placed on the device read only tab, so that they cannot edit, delete or modify mainly the imei of the device, it is very rare leave a normal user without that box, but that limits that option to change the name, maybe a more intermediate permission category could be created? thank you
